src/shared/constants/index.ts

```diff
@@ -168,3 +168,8 @@ export const PAYG_MAX_CREDIT = 250
 export const PAYG_CREDIT_EXPERIMENT_ID = process.env.PAYG_CREDIT_EXPERIMENT_ID
 export const CREDIT_250_EXPERIMENT_ID = 'JVfGg-umSiSk-JVpeh4hGA'
 export const CREDIT_250_EXPERIENCE_CUTOFF_EPOCH = 1646697600000 // 8 March 2022
+
+export const INFLUXDATA_SUPPORT_CONTACT_UK = '+44 808-196-6148'
+export const INFLUXDATA_SUPPORT_CONTACT_UK_NO_SPACE = 'tel:+448081966148'
```

Comment:
nitpick: `NO_SPACE` might be more descriptive if it's called `TELEPHONE_LINK` since that's what the `tel:` prefix is. so like 

```ts
INFLUXDATA_SUPPORT_CONTACT_UK_TELEPHONE_LINK
INFLUXDATA_SUPPORT_CONTACT_US_TELEPHONE_LINK
```